packages/accounts-base/accounts_common.js

```diff
@@ -19,6 +19,7 @@ const VALID_CONFIG_KEYS = [
   'loginTokenExpirationHours',
   'tokenSequenceLength',
   'clientStorage',
+  "ddpUrl"
```

Comment:
Please follow the format of using `'` instead of `"` as above.

Reply:
You are right